.bazelci/presubmit.yml

```diff
@@ -2,22 +2,31 @@
 platforms:
   ubuntu1604:
     build_targets:
-    - "..."
+    - "//..."
     test_targets:
-    - "..."
+      - "--"
+      - "//..."
+      - "-//javatests/build/bazel/tests/integration:RBEExampleTest/bazel0.22.0"
```

Comment:
Could you leave a comment explaining why this version is excluded. I assume it's because RBE wasn't complete till 0.24.x, but am not sure.

Reply:
Not sure I understand what you mean.
This version isn't excluded but rather all instances of `RBEExampleTest` are excluded. It has two instances (22/23) because of toolchains issue.
It's excluded because this test can only be run on RBE.
The real solution is to bake RBE support into the library (for example internally we use a define to say if it's using RE or not).
A better solution would have been to tag this test as something and excluded these kind of tests but due to #85 that isn't an option right now.
Does that answer the question?

Reply:
I'll merge this as is and I'll just try to fix #85 sometime this week. 
Actual fix is really small, test will take a bit more.
Once that's done I'll tag the RBE test and it will be clearer.